packages/rs-dpp/src/data_trigger/mod.rs

```diff
@@ -97,14 +98,12 @@ impl DataTrigger {
 
         match maybe_execution_result {
             Err(err) => {
-                let consensus_error = DataTriggerError::DataTriggerExecutionError {
+                let consensus_error = DataTriggerExecutionError::new(
                     data_contract_id,
-                    document_transition_id: *get_from_transition!(document_transition, id),
-                    message: err.to_string(),
-                    execution_error: err,
-                    document_transition: None,
-                    owner_id: None,
-                };
+                    *get_from_transition!(document_transition, id),
```

Comment:
What's this macro doing? I didn't stumble upon it yet, does it just gets a field value out of the ST?

Reply:
We didn't touch this thing in the PR, but according to the code it just takes a reference to the field from document transition.